app/assets/javascripts/discourse/controllers/insert-hyperlink.js.es6

```diff
@@ -0,0 +1,50 @@
+import ModalFunctionality from "discourse/mixins/modal-functionality";
+import { default as computed } from "ember-addons/ember-computed-decorators";
+
+export default Ember.Controller.extend(ModalFunctionality, {
+  linkUrl: "",
+  linkText: "",
+
+  @computed()
+  title: () => "composer.link_dialog_title",
```

Comment:
Does it have to be a computed property if the value is static?

Reply:
Good spot, no, it doesn't need to be computed at all.